relayer-cli/src/commands/query/packet.rs

```diff
@@ -46,7 +43,7 @@ impl QueryPacketCommitmentsCmd {
         let opts = QueryPacketOptions {
             port_id: self.port_id.clone(),
             channel_id: self.channel_id.clone(),
-            height: self.height.unwrap_or(0_u64),
+            height: 0_u64,
```

Comment:
Should we remove this field as well or is it actually required but never used?

Reply:
Doh! Actually I didn't mean to remove these for the proof queries. Added back, thanks!